credentialdigger/cli/get_discoveries.py

```diff
@@ -0,0 +1,279 @@
+"""
+The 'get_discoveries' module can be used to retrieve the discoveries on the fly
+from the terminal. It supports both the Sqlite and Postgres clients.
+
+NOTE: Postgres is used by default. Please make sure that the environment
+variables are exported.
+
+usage: credentialdigger get_discoveries [-h] [--dotenv DOTENV]
+                                        [--sqlite SQLITE]
+                                        [--save SAVE]
+                                        repo_url
```

Comment:
from below I see there are 2 more options that you haven't listed here: filename and state